src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -3645,4 +3645,175 @@ public Long hstrlen(final String key, final String field) {
     return client.getIntegerReply();
   }
 
+  @Override
+  public EntryID xadd(final String key, final EntryID id, final Map<String, String> hash) {
+    return xadd(key, id, hash, Long.MAX_VALUE, true);
```

Comment:
In case on MAX_VALUE the MAXLEN is not serialized on the command call. see: https://github.com/gkorland/jedis/blob/d7fceb249ec1982d7727a4597de427da052c4356/src/main/java/redis/clients/jedis/BinaryClient.java#L1263

If the user wants to set the MAXLEN he can call the overloaded xadd: https://github.com/gkorland/jedis/blob/d7fceb249ec1982d7727a4597de427da052c4356/src/main/java/redis/clients/jedis/Jedis.java#L3654